guides/core_concepts/0020-pipeline-editor.md

```diff
@@ -0,0 +1,69 @@
+# Pipeline Editor
+
+The Pipeline Editor is a visual tool designed to facilitate the construction of [Pipelines](0004-pipelines.html).
+It is integrated in Astarte Dashboard and automatically opens up when you head to create a new pipeline.
+
+## Overview
+
+In Astarte Dashboard, go to the Pipelines section and head to create a new pipeline.
+You should now be able to see the visual editor, below the field to specify the pipeline's name.
+
+The visual editor itself is composed of two parts:
+
+- a sidebar listing all available [Blocks](0003-blocks.html), grouped together by type
+- an empty space where you can drag & drop blocks to, then connecting them together to effectively design a pipeline
```

Comment:
Small rephrase:
an empty space where you can drag & drop blocks , connecting them together to effectively design a pipeline